### PR TITLE
[FSTORE-36] Update val_size

### PR DESCRIPTION
**Pull request by kennethmhc**

### CODE REVIEW

**Review by jimdowling — APPROVED** [review]

File: docs/user_guides/fs/feature_view/training-data.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ version, job = feature_view.create_train_test_split(
 Create a training dataset (as in-memory DataFrames) or materialise a training dataset with train, validation, and test splits.
 ```python
 # create a training dataset as DataFrame
-X_train, X_val, X_test, y_train, y_val, y_test = feature_view.train_validation_test_splits(val_size=0.3, test_size=0.2)
+X_train, X_val, X_test, y_train, y_val, y_test = feature_view.train_validation_test_splits(validation_size=0.3, test_size=0.2)
```
Comment (SirOibaf) [inline docs/user_guides/fs/feature_view/training-data.md:63]: The method name here is still wrong, it's `train_validation_test_split` without the (s)

File: docs/user_guides/fs/feature_view/training-data.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ version, job = feature_view.create_train_test_split(
...
 
 # materialise a training dataset
 version, job = feature_view.create_train_validation_test_splits(
```
Comment (SirOibaf) [inline docs/user_guides/fs/feature_view/training-data.md:66]: Same comment on the method name.

**Review by SirOibaf — CHANGES_REQUESTED** [review]
@kennethmhc - Beside the one I pointed out, can you make sure we don't have other `splits` left?

**Comment by kennethmhc** [comment]
@SirOibaf I made a search in logicalclocks org, cannot find any more.

**Review by SirOibaf — APPROVED** [review]

**Review by jimdowling — APPROVED** [review]

**Pull request merged**